tensorflow_federated/python/core/impl/executors/sizing_executor_test.py

```diff
@@ -88,6 +88,45 @@ async def _make():
     self.assertCountEqual(ex.broadcast_history, [[10, tf.int32]])
     self.assertCountEqual(ex.aggregate_history, [[1, tf.int32]])
 
+  def test_unknown_input_type(self):
+    ex = sizing_executor.SizingExecutor(eager_tf_executor.EagerTFExecutor())
+
+    tensor_type = computation_types.TensorType(tf.int32, [None])
+
+    @computations.tf_computation(tensor_type)
+    def return_constant(x):
+      del x
+      return tf.constant(0)
+
+    async def _make():
+      v1 = await ex.create_value(return_constant)
+      v2 = await ex.create_value([1, 2, 3], tensor_type)
+      v3 = await ex.create_call(v1, v2)
+      return await v3.compute()
+
+    asyncio.get_event_loop().run_until_complete(_make())
+    self.assertCountEqual(ex.broadcast_history, [[3, tf.int32]])
+    self.assertCountEqual(ex.aggregate_history, [[1, tf.int32]])
+
+  def test_unknown_output_type(self):
+    ex = sizing_executor.SizingExecutor(eager_tf_executor.EagerTFExecutor())
+
+    tensor_type = computation_types.TensorType(tf.int32, None)
+
+    @computations.tf_computation(tensor_type)
+    def return_constant(x):
+      return tf.range(x)
+
+    async def _make():
+      v1 = await ex.create_value(return_constant)
+      v2 = await ex.create_value([3], tensor_type)
```

Comment:
It looks like this line is failing automated testing with 

```
TypeError: The apparent type int32[1] of a tensor [3] does not match the expected type int32.
```

I suspect `tensor_type` may need to have a shape `[None]`, or the value here needs to be `3`?

Reply:
resolved, it was commited by accident the intention was `ex.create_value(3, tensor_type)`